programs/clearing_house/src/math/oracle.rs

```diff
@@ -27,6 +51,7 @@ pub struct OracleStatus {
     pub oracle_mark_spread_pct: i128,
     pub is_valid: bool,
```

Comment:
do we still need is_valid? => could change it to a method is_valid() which checks == Valid